app/src/main/java/com/qulix/shestakaa/gifsearchermvp/view/RecyclerAdapter.java

```diff
@@ -21,7 +21,7 @@
 
 class RecyclerAdapter extends RecyclerView.Adapter<RecyclerAdapter.MyViewHolder> {
 
-    private List<Data> mData;
+    private final List<Data> mData;
     private final Context mContext;
     private Presenter mPresenter;
```

Comment:
Need to change code somehow to not to store presenter here. I tried to pass onClickListener from View into the adapter, but in such case I can't return gifUrl back.